elastalert/elastalert.py

```diff
@@ -967,6 +967,10 @@ def set_realert(self, rule_name, timestamp, exponent):
 
     def is_silenced(self, rule_name):
         """ Checks if rule_name is currently silenced. Returns false on exception. """
+
+        if self.debug:
```

Comment:
I think this should go below the check for silence_cache. Debug should still have normal realert behavior, minus the fetching from Elasticsearch part.